hazenlib/spatial_resolution.py

```diff
@@ -405,7 +408,7 @@ def get_esf(edge_arr, y):
     return u, esf
 
 
-def calculate_mtf_for_edge(dicom, edge, report_path=False):
+def calculate_mtf_for_edge(dicom, edge, report_path=True):
```

Comment:
Is there a reason that this is changed now? I think we want to keep the report_path kwarg default as False.

Reply:
hey yes sorry I did it for debugging. So, should I put this back to how it was?

Reply:
yep 👍🏽